graphql_api/mutation/information_source.py

```diff
@@ -91,6 +91,23 @@ def mutate(self, info, project_id: str, value: bool):
         return ToggleInformationSource(ok=True)
 
 
+class SetAllModelCallbacksSelected(graphene.Mutation):
+    class Arguments:
+        project_id = graphene.ID(required=True)
+        value = graphene.Boolean(required=True)
+
+    ok = graphene.Boolean()
+
+    def mutate(self, info, project_id: str, value: bool):
+        auth.check_demo_access(info)
+        auth.check_project_access(info, project_id)
+        manager.set_all_model_callbacks_selected(project_id, value)
+        notification.send_organization_update(
+            project_id, f"information_source_updated:all"
```

Comment:
is this correct? (information_source_update) this will trigger an update on the heuristics page as well

Reply:
No, only meant as an update for the model callbacks. i commented it out